manifests/c/CoreFTP/FTPServer/2.0-749/CoreFTP.FTPServer.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: CoreFTP.FTPServer
+PackageVersion: 2.0-749
+PackageLocale: en-US
+Publisher: coreFTP
+PublisherUrl: https://www.coreftp.com/
+PackageName: Core FTP / SFTP Server

```

```suggestion
PackageName: Core FTP Server
```
This should match control panel